br/pkg/restore/split/split_client.go

```diff
@@ -1,11 +1,12 @@
 // Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
 
-package restore
+package split
```

Comment:
shall we change the file name from `split_client` to `client`.

Reply:
Done